### DISCUSSION TITLE
Show Aircarft Flyover

### DISCUSSION THREAD

**Discussion by Ze-Astro-Boy** [Stellarium/stellarium#2512] [Ideas]
A potentially useful tool that could be implemented is using live flight data used by sites such as FlightAware to track aircraft flying over. This would only be in use when the time is set to live or previous as the flight of aircraft is a very variable event, and there is a network connection to fetch the live data.

**Comment by alex-w** [comment]
Anyone can update the Planes plugin: https://github.com/Stellarium/stellarium/tree/planes/plugins/Planes

**Reply by Ze-Astro-Boy** [reply]
I looked and did not see that this idea was already considered. Thanks for directing me towards it!
